runtime/compiler/il/J9Node.cpp

```diff
@@ -2212,6 +2212,23 @@ J9::Node::setDontInlinePutOrderedCall()
    }
 
 bool
+J9::Node::isUnsafeCopyMemoryIntrinsic()
+   {
+   if (!self()->getOpCode().isCall() || !self()->getSymbol()->isMethod())
+      return false;
+   TR::MethodSymbol *symbol = self()->getSymbol()->getMethodSymbol();
+   if (!symbol || !symbol->isNative())
+      return false;
+   switch (symbol->getRecognizedMethod())
+      {
+      case TR::sun_misc_Unsafe_copyMemory:
+      case TR::jdk_internal_misc_Unsafe_copyMemory0:
+         return true;
+      default:
+         return false;
+      }
```

Comment:
So many return points! Now sure if this would be cleaner:

```
   if (self()->getOpCode().isCall() && self()->getSymbol()->isMethod())
      {
      TR::MethodSymbol *symbol = self()->getSymbol()->getMethodSymbol();
      if (symbol != NULL && symbol->isNative())
         {
         switch (symbol->getRecognizedMethod())
            {
            case TR::sun_misc_Unsafe_copyMemory:
            case TR::jdk_internal_misc_Unsafe_copyMemory0:
                return true;
            default:
                break;
            }
         }
      }
   
   return false;
```

Reply:
Fixed in https://github.com/eclipse/openj9/compare/5b7e35a24c88400aebba54647550e37df536c44c..91cf3d3105d3cec00915a532186cdcbf5977262d